Tensile/Tests/extended/direct_to_lds/dtl_sgemm.yaml

```diff
@@ -0,0 +1,1021 @@
+TestParameters:
+  marks: [skip-gfx900, skip-gfx906, skip-gfx1010, skip-gfx1011, skip-gfx1012, skip-gfx1030] # not supported by arch
+
+GlobalParameters:
+  NumElementsToValidate: -1
+  BoundsCheck: True
+  KernelTime: True
+  # PrintSolutionRejectionReason: True
+
+BenchmarkProblems:
+  ########################################
+  # NN
+  ########################################
+  - # sgemm NN
+    - # ProblemType
+      OperationType: GEMM
+      DataType: s
+      TransposeA: False
+      TransposeB: False
+      UseBeta: True
+      Batched: True
+
+    - # VW = 2
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - KernelLanguage: ["Assembly"]
+        - EdgeType: ["ShiftPtr"]
+        - AtomicAddC: [False] 
+      ForkParameters:
+        - MatrixInstruction:
+          # - [16, 16, 4, 1, 1, 1,4, 4,1]  # 64x64
+          - [16, 16, 4, 1, 1, 2,2, 2,2]  # 64x64
+          # - [16, 16, 4, 1, 1, 4,1, 1,4]  # 64x64
+          # - [16, 16, 4, 1, 1, 3,2, 2,2]  # 96x64 # MT must be power of 2
+          # - [16, 16, 4, 1, 1, 2,3, 2,2]  # 64x96 # MT must be power of 2
+          - [16, 16, 4, 1, 1, 2,4, 4,1]  # 128x64
+          - [16, 16, 4, 1, 1, 4,2, 2,2]  # 128x64
+          # - [16, 16, 4, 1, 1, 8,1, 1,4]  # 128x64
+          # - [16, 16, 4, 1, 1, 1,8, 4,1]  # 64x128
+          - [16, 16, 4, 1, 1, 2,4, 2,2]  # 64x128
+          - [16, 16, 4, 1, 1, 4,2, 1,4]  # 64x128
+          - [16, 16, 4, 1, 1, 2,8, 4,1]  # 128x128
+          - [16, 16, 4, 1, 1, 4,4, 2,2]  # 128x128
+          - [16, 16, 4, 1, 1, 8,2, 1,4]  # 128x128
+          - [16, 16, 4, 1, 1, 4,4, 4,1]  # 256x64
+          - [16, 16, 4, 1, 1, 4,4, 1,4]  # 64x256
+          - [16, 16, 4, 1, 1, 4,4, 2,2]  # 128x128
+        - ThreadTile:
+          - [  8, 32 ]
+        - WorkGroup:
+          - [ 16, 16, 1 ]
+        - SourceSwap: [True]
+        - PrefetchGlobalRead: [1] # [1, 2]
+        - AssertFree0ElementMultiple: [2]
+        # - AssertFree1ElementMultiple: [2]
+        - WorkGroupMapping: [8]
+        - PrefetchLocalRead: [5]
+        - GlobalSplitU: [1]
+        - DepthU:  [16]
+        - StoreVectorWidth: [2]
+        - VectorWidth: [2]
+        - GlobalReadVectorWidth: [1, 2]
+        - LocalReadVectorWidth: [1] # [1, 2]
+        - DirectToLds: [True]
+        - DirectToVgprA: [False] # DirectToVgpr only for dgemm right now
+        - DirectToVgprB: [False] # DirectToVgpr only for dgemm right now
+        - WaveSeparateGlobalReadA: [0]
+        - WaveSeparateGlobalReadB: [0]
+        - NumLoadsCoalescedA: [1] # NLC=2 not working
+        - NumLoadsCoalescedB: [1] # NLC=2 not working
+        - ScheduleIterAlg: [3]
+        - AssertSummationElementMultiple: [32]
+        - StaggerU: [0]
+        - NumElementsPerBatchStore: [0] # minimum 2 for SCIU
+        - FractionalLoad: [0, 1, 2]
+        - BufferLoad: [True, False]
+        - TransposeLDS: [0, 1]
```

Comment:
TransposeLDS=0 should not be necessary for DTL test.
DTL works only with TransposeLDS=1 for TLU=false case.